Solutions/Tomcat/Data Connectors/Connector_Tomcat_agent.json

```diff
@@ -2,7 +2,7 @@
     "id": "ApacheTomcat",
     "title": "Apache Tomcat",
     "publisher": "Apache",
-    "descriptionMarkdown": "The Apache Tomcat data connector provides the capability to ingest [Apache Tomcat](http://tomcat.apache.org/) events into Azure Sentinel. Refer to [Apache Tomcat documentation](http://tomcat.apache.org/tomcat-10.0-doc/logging.html) for more information.",
+    "descriptionMarkdown": "The Apache Tomcat data connector provides the capability to ingest [Apache Tomcat](http://tomcat.apache.org/) events into Microsoft Sentinel. Refer to [Apache Tomcat documentation](http://tomcat.apache.org/tomcat-10.0-doc/logging.html) for more information.",
```

Comment:
Please check if we need to replace `data connector` text with `solution`
![image](https://user-images.githubusercontent.com/107389644/175493200-661d7851-4f0f-40cc-ab2b-c09d34ece96b.png)